### ISSUE TITLE
Tiles stacking

### ISSUE THREAD

**Issue by cyllandra** [Fugiman/google-meet-grid-view#337]
When host is presenting, the grid view defaults to stacked view. Sometimes restarting the meet fixes this problem. This is a new problem, it was previously fixed, but has been noticed as of February 11, 2021.

**Comment by BlamedPlace** [comment]
I have stacked view even if not presenting. Started about a week ago

**Comment by iamamy20883** [comment]
YES! The stacked view is VERY frustrating! We are not able to see our whole class and its more of a liability then anything! Please fix this ASAP!

**Comment by lakenkraft** [comment]
Tiles will not stop stacking! I have tried uninstalling grid view, I have tried other grid view programs and NONE of them will allow me to see all of the participants at once. It's driving me insane. Please fix this!

**Comment by FoxForFriends** [comment]
Have same issue, needs to be fixed asap please!

**Comment by ITeach2ndGrade** [comment]
Please help! I am having the same issue. I deleted the Google Meet Grid View (Fix) extension hoping that reinstalling it would fix the problem, however, the "Fix" extension is no longer available. PLEASE help! I'm online with my class ALL DAY, EVERY DAY, and I need to see my students while I'm teaching.

**Comment by Daverweisson** [comment]
I am having the same problem please fix the issue, let us know the solution

**Comment by Eagle-Man** [comment]
This extension has long been abandoned.  From the Grid View Fix GitHub page:

> UPDATE 02/11/2021:
It turns out that Google is testing an update of Meet that disables the Grid View extension. This is why sometimes it works and sometimes not (you see the boxes in a vertical bar). If you seriously need Grid View right now, you might consider refreshing a few times the browser tab (you should end up loading the old compatible version at a certain point).
I am really sorry but I have to temporarily remove the extension from Chrome Web Store. In fact, updating Grid View would require a huge amount of work or may not be feasible at all. Please, remember that Google Meet introduced native grid view (up to 49 tiles).

**Comment by Jkarhani** [comment]
Please update it .. or get us an alternative.. it's chaos..

**Comment by Eagle-Man** [comment]
Sorry.  While it may not be your intention, this comes across as a demand.  A third (or fourth) party developer who does this in their free time with no compensation does not owe anyone anything.  They do not have to continue updates if they no longer have the time or desire, and they do not owe an alternative to their users.  If you have issues with Meet, take it up with Google.
